cmd/podman/build.go

```diff
@@ -225,6 +225,14 @@ func buildCmd(c *cliconfig.BuildValues) error {
 	for _, arg := range c.RuntimeFlags {
 		runtimeFlags = append(runtimeFlags, "--"+arg)
 	}
+
+	conf, err := runtime.GetConfig()
+	if err != nil {
+		return err
+	}
+	if conf.CgroupManager == "systemd" {
+		runtimeFlags = append(runtimeFlags, "--systemd")
+	}
```

Comment:
I'm not seeing this `--systemd` flag in the `buildah bud` or the `podman build` man pages.  Should it be added?

Reply:
this option is passed down to the OCI runtime AFAICS, so Buildah doesn't need to know about it

Reply:
OK, let me reask.  Would a user ever have to type `--systemd` ?  If so, then we need to document it in a man page somewhere.  If it's all internal, then no need.

Reply:
We have not had to deal with it yet, but I think adding this to buildah would be useful.

Reply:
an user could force `--runtime-flag systemd` for buildah but I think that is an internal detail and the current documentation covers it